README.md

```diff
@@ -1 +1,57 @@
-# E-commerce-app-team0
+
+# E-commerce-app-Team0
+
+### Why we did this? 
+- This app was designed and implemented to make people's lifes easier selling and buying things online. 
+- This is the final project from the Fundamentals program in web development provided by Gaza Sky Geeks (GSG). 
+- It aims at developing our web development skills and gaining new skills other than those acquired during the program. 
+---
+### What is this app?
+- It is a platform for selling and buying products
+---
+### User Story 
+- In this website there are two types of users: **Seller** & **Customer/Buyer**
+
+- The seller can do the following: 
+    - Add a product.
+    - See all the products of his/her own. 
+    - Search through his/her list of products.
+    - Delete or Edit any product of his/her own.
+    
+- The Buyer can do this: 
+    1. See all items available on the store.
+    2. Search for an item by its name. 
+    3. Filter items by category or sort them by price. 
+    4. Add items to or delete them from his/her cart. 
+    5. See all items in his/her own cart alongside their total price. 
+    6. Display items in his/her cart either as a grid or list
+---
+
+### User Journey 
+- This app provides ways for: 
+    1. **sellers** to enlist products they want to sell. 
+    2. **Customers or buyers** can see the list of products available and choose what to buy. 
+---
+
+### Technologies used 
+- Programming languages: 
+    - HTML 
+    - CSS 
+    - JavaScript 
+- Integrated Development Environment (IDE): 
+    - Visual Studio Code 
+ 
+- Code Versioning Tool: 
+    - Git using Github   
+---
+
+### Team Memebers: 
+- Hamzah 
```

Comment:
you can add your Github profile links, for example:
![image](https://user-images.githubusercontent.com/54964739/99385316-7b810780-28d9-11eb-8a0b-1a1c3368b9ba.png)

Reply:
Thank you